ifbcat_api/management/commands/load_keywords_translate_from_csv.py

```diff
@@ -16,33 +16,30 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         if not os.path.exists(options["file"]):
-            logger.error(f"Cannot find the file named {options['file'][12:]}")
-        else:
-            with open(os.path.join(options["file"]), encoding='utf-8') as data_file:
-                data = csv.DictReader(data_file, delimiter='\t')
-                if 'French_keywords' and 'English_keywords' in data.fieldnames:
-                    count = 0
-                    untranslated = 0
-                    qs_dict = Keyword.objects.all().values().order_by('keyword')
-                    file_dict = list(data)
+            return logger.error(f"Cannot find the file named {options['file'][12:]}")
+        with open(os.path.join(options["file"]), encoding='utf-8') as data_file:
+            data = csv.DictReader(data_file, delimiter='\t')
+            if 'French_keywords' and 'English_keywords' in data.fieldnames:
+                count = 0
+                untranslated = 0
```

Comment:
Starting here I haven't reviewed, I will wait for the previous comment to be resolved